### PR TITLE
Add Redirection After Account Creation

### LINKED ISSUE #31: Redirect to accounts list or account details page after creating a new account
Currently, when we create a new account, we are redirected on the same form after submitting it. We should be redirected on the accounts list, or the detail page of the account just created.

### PR DESCRIPTION
**Pull request by GuenoleCHERUBIN**
Resolves #31

### CODE REVIEW

File: controllers/ManagementController.php [hunk 1]
```diff
@@ -31,6 +31,7 @@ public function actionAccounts($id = null)
         }
         $usersDataProvider = new ActiveDataProvider([
             'query' => User::find(),
+            'sort' => ['defaultOrder'=>['id' => SORT_DESC]],
```
Comment (EmileCalixte) [inline controllers/ManagementController.php]: Spaces missing around arrow

File: controllers/ManagementController.php [hunk 2]
```diff
@@ -195,7 +196,9 @@ private function createAccount(CreateAccountForm $form)
         $user->is_active = true;
         $user->role         = $form->role;
```
Comment (EmileCalixte) [inline controllers/ManagementController.php]: Why so much spaces ?

**Review by EmileCalixte — CHANGES_REQUESTED** [review]

File: controllers/ManagementController.php [hunk 3]
```diff
@@ -195,7 +196,9 @@ private function createAccount(CreateAccountForm $form)
...
 
-        if(!$user->save()){
+        if($user->save()){
```
Comment (alexisljn) [inline controllers/ManagementController.php]: You don't need the if else structure, just make ```if(!$user->save()) {
throw new CannotSaveException($user) 
}```
and then you redirect to the account newly created

**Review by alexisljn — CHANGES_REQUESTED** [review]

File: controllers/ManagementController.php (hunk 3 above)
Comment (EmileCalixte) [reply controllers/ManagementController.php]: I agree !

**Review by EmileCalixte — APPROVED** [review]

**Review by alexisljn — APPROVED** [review]

**Pull request merged**